OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -52,6 +52,9 @@ public class ExplodesInfo : ConditionalTraitInfo, Requires<IHealthInfo>
 			"Footprint (explosion on each occupied cell).")]
 		public readonly ExplosionType Type = ExplosionType.CenterPosition;
 
+		[Desc("Offset of the explosion.")]
```

Comment:
```suggestion
		[Desc("Offset of the explosion from the center of the cell.")]
```

Reply:
That isn't entirely correct for footprint based explosions.

Reply:
```
weapon.Impact(Target.FromPos(self.World.Map.CenterOfCell(c) + Info.Offset), source);
```

```
weapon.Impact(Target.FromPos(self.CenterPosition + Info.Offset), source);
```

seems correct to me

Reply:
Well, I guess the edit is fine.

Reply:
Updated.